feat: cli documenation (#35)
NoumenaDigital/npl-cli: README.md

# NPL CLI

> To check full CLI documentation go to: [cli.md](docs/runtime/tools/build-tools/cli.md)

Reviewer: ```suggestion
You can find the full CLI documentation here: [cli.md](docs/runtime/tools/build-tools/cli.md)
```
